packages/doenetml-worker-rust/lib-doenetml-core/src/core/general_state_var/string_state_var.rs

```diff
@@ -158,3 +158,273 @@ impl StateVarUpdater<String, StringRequiredData> for StringStateVar {
         }
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    #[test]
+    fn string_state_var_with_no_dependencies() {
```

Comment:
It would be nice to have some docstrings that had some example doenetml where such a state variable would arise.